Beginnings of Sale page (#13785)
fleetdm/fleet: handbook/customers/README.md

# Customers

> This handbook page details processes specific to working [with](fleetdm.com/handbook/company/customers#contact-us) and [within](fleetdm.com/handbook/company/customers#responsibilities) the Sales department.

Reviewer: ```suggestion
This handbook page details processes specific to working [with](#contact-us) and [within](#responsibilities) this department.
```

|:------------------------------|:------------------------------------------------------------------------------------------------------------------------|
| Chief Revenue Officer               | [Alex Mitchell](https://www.linkedin.com/in/alexandercmitchell/) _([@alexmitchelliii](https://github.com/alexmitchelliii))_
| VP of Customer Success              | [Zay Hanlon](https://www.linkedin.com/in/zayhanlon/) _([@zayhanlon](https://github.com/zayhanlon))_
| Director of Solutions Consulting    | [Dave Herder](https://www.linkedin.com/in/daveherder/) _([@dherder](https://github.com/dherder))_
| Head of Public Sector               | [Keith Barnes](https://www.linkedin.com/in/keith-barnes-8b666/) _([@KAB703](https://github.com/KAB703))_
| Business Expansion Lead             | [Tom Ostertag](https://www.linkedin.com/in/tom-ostertag-77212791/) _([@TomOstertag](https://github.com/TomOstertag))_
| Lead Sales Development Rep          | [Brad Macdowall](https://www.linkedin.com/in/bradmacdowall/) _([@BradMacd](https://github.com/BradMacd))_
| TODO FINISH          | [Brad Macdowall](https://www.linkedin.com/in/bradmacdowall/) _([@BradMacd](https://github.com/BradMacd))_

## Contact us

> - To make a request of this department, [create an issue](https://github.com/fleetdm/confidential/issues/new?assignees=&labels=%23g-marketing&projects=&template=custom-request.md&title=Request%3A+_______________________) and a team member will get back to you within one business day.

Reviewer: wrong link (see `labels=`)

>   - Any Fleet team member can [view the kanban board](https://app.zenhub.com/workspaces/g-marketing-64e6c8e2d35c7f001a457b7f/board) for this department, including pending tasks in the active sprint and the status of new requests.

Reviewer: wrong board

  - Please use issue comments and GitHub mentions to communicate about your request, including follow-ups.  If you mention a team member in a GitHub comment, you will get a reply within one business day.

> - To suggest an idea for social media, an article, or a video, [create an issue (#g-marketing)](https://github.com/fleetdm/confidential/issues/new?assignees=&labels=%23g-marketing&projects=&template=custom-request.md&title=Request%3A+_______________________) and concisely suggest it.

Reviewer: ```suggestion

```

> - If urgent, mention a marketing [team member](#team) in the [#g-marketing](https://fleetdm.slack.com/archives/C01ALP02RB5) Slack channel:

Reviewer: ```suggestion
- If urgent, mention a [team member](#team) in the [#g-sales](https://fleetdm.slack.com/archives/TODO) Slack channel:
```

>   - For all urgent questions related to events, video, or social media, mention the Field Marketer.

Reviewer: ```suggestion

```

>   - For all urgent questions related to articles or the Fleet community, mention the Community Advocate.

Reviewer: ```suggestion

```
